src/notes.tsx

```diff
@@ -70,6 +73,8 @@ const Notes = () => {
   const syncRef = useRef<Sync | undefined>(undefined);
   syncRef.current = sync;
 
+  const [initialized, setInitialized] = useState<boolean>(false);
```

Comment:
If the application is initialized, and there are no existing notes, New note modal will be presented with no Cancel option.